pkg/minikube/cruntime/docker.go

```diff
@@ -252,3 +265,34 @@ func (r *Docker) ContainerLogCmd(id string, len int, follow bool) string {
 func (r *Docker) SystemLogCmd(len int) string {
 	return fmt.Sprintf("sudo journalctl -u docker -n %d", len)
 }
+
+// Preload preloads docker with k8s images:
+// 1. Copy over the preloaded tarball into the VM
+// 2. Extract the preloaded tarball to the correct directory
+// 3. Remove the tarball within the VM
+func (r *Docker) Preload(k8sVersion string) error {
+	tarballPath := download.TarballPath(k8sVersion)
+	dest := "/tmp/preloaded.tar"
+
+	// Copy over tarball into host
+	fa, err := assets.NewFileAsset(tarballPath, filepath.Dir(dest), filepath.Base(dest), "0644")
+	if err != nil {
+		return errors.Wrap(err, "getting file asset")
+	}
+	t := time.Now()
+	if err := r.Runner.Copy(fa); err != nil {
+		return errors.Wrap(err, "copying file")
+	}
+	glog.Infof("Took %f seconds to copy over tarball", time.Since(t).Seconds())
+
+	// extract the tarball to /var in the VM
+	if rr, err := r.Runner.RunCmd(exec.Command("sudo", "tar", "-I", "lz4", "-C", "/var", "-xvf", dest)); err != nil {
+		return errors.Wrapf(err, "extracting tarball: %s", rr.Output())
+	}
+
+	//  remove the tarball in the VM
+	if rr, err := r.Runner.RunCmd(exec.Command("sudo", "rm", dest)); err != nil {
```

Comment:
sudo shouldn't be required - but I believe you can just call `r.Runner.Remove(fa)` as well.